Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/Controller.purs

```diff
@@ -151,6 +205,22 @@ eval (APIFailureActionController (ErrorModal.PrimaryButtonActionController Prima
 
 eval (NoRidesActionController (ErrorModal.PrimaryButtonActionController PrimaryButton.OnClick)) state = exit GoHome
 
+eval (EmailEditTextAC (PrimaryEditText.TextChanged id a)) state = continue state{data {email = trim(a)},props{btnActive = if ((length (trim(a)) > 0 ) && (length state.data.description > 2) && (validateEmail a)) then true else false}}
```

Comment:
Instead of writing the continue then true false we can simply use the condition itself. And also remove unwanted brackets